onnx_xla/operator_registry.cc

```diff
@@ -34,4 +34,20 @@ TranslationMap& OperatorRegistry::map() {
   static TranslationMap map;
   return map;
 }
+
+XlaComputation OperatorRegistry::add(PrimitiveType dataType) {
+  XlaBuilder builder("add");
+  auto y = builder.Parameter(0, ShapeUtil::MakeShape(dataType, {}), "y");
+  auto x = builder.Parameter(1, ShapeUtil::MakeShape(dataType, {}), "x");
+  builder.Add(y, x);
+  return builder.Build().ConsumeValueOrDie();
```

Comment:
Noob question: what does `ConsumeValueOrDie` do?

Reply:
It takes a StatusOr object, checks the status is ok, and returns the object itself

Reply:
Hmm, what does the `Die` part do?

Reply:
Tensorflow throws its error explaining why the computation couldn't be built. As an example, I intentionally screwed up broadcast dimensions in the sum operator translation and got this message `Invalid argument: Broadcast dimensions field must either be not set or be the identity on binary operations with operands of the same rank. Aborted`

Reply:
No, it's OK. I was just curious about its behavior.